lib/custom_lib/custom_lib.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021, Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* The C preprocessor namespace is global, so Zephyr regularly
+ * uses encoded pathnames in the header file multiple-inclusion guards.
+ * For example:
+ *
+ *   zephyr/lib/libc/minimal/include/sys/timespec.h uses
+ *   ZEPHYR_LIB_LIBC_MINIMAL_INCLUDE_SYS_TIMESPEC_H_
+ */
+
+#ifndef EXAMPLE_APPLICATION_LIB_CUSTOM_LIB_CUSTOM_LIB_H_
```

Comment:
```suggestion
#ifndef EXAMPLE_APPLICATION_INCLUDE_CUSTOM_LIB_CUSTOM_LIB_H_
```

Reply:
Accepted, and rolled in (along with relocating the file.